docs/features/feature-guides/compliance-forms/create-a-form.md

```diff
@@ -0,0 +1,159 @@
+4. Preview the relevant Assets to confirm you have applied the appropriate filters
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-assign-assets.png"
+       alt="Assign assets to a Compliance Form"/>
+</p>
+
+### Step 4: Add Recipients to your Form
+
+Now that you have defined the set of Questions to be answered for a set of Assets, it's now time to delegate out to your Users.
+
+In the **Add Recipients** section, decide who is responsible for completing the Form:
+
+* **Asset Owners:** Any User that is assigned to one of the in-scope Assets will be able to complete the Form. This is useful for larger initiatives when you may not know the full set of Users.
+* **Specific Users and/or Groups:** Select a specific set of Users and/or Groups within DataHub. This is useful when Ownership of the Assets may be poorly-defined.
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-add-users-or-groups.png"
+       alt="Assign recipients to a Compliance Form"/>
+</p>
+
+### Step 5: Publish your Form
+
+Once you have defined the set of Questions to be completed, the in-scope Assets, and the relevant Recipients, click **Publish** and Users will be able to complete your Form!
+
+:::caution
+Once you have published a Form, you **cannot** change or add Questions. You can, however, change the set of Assets and/or Assignees for the Form.
+:::
+
+Not ready for primetime just yet? No worries! You also have the option to **Save Draft**.
+
+<p align="center">
+  <img
+       width="80%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/create-compliance-form-publish.png"
+       alt="Publish a Compliance Form"/>
+</p>
+
+## FAQ and Troubleshooting
+
+**How does a Compliance Form interact with existing metadata?**
+
+If an asset already has existing metadata that is also referenced in a Form Question, users assigned to the Form will have the option to confirm the value and make no changes, overwrite the value, or append additional details.
+
+**What is the difference between Completion and Verification Forms?**
+
+Both form types are a way to configure a set of optional and/or required questions for DataHub users to complete. When using Verification Forms, users will be presented with a final verification step once all required questions have been completed; you can think of this as a final acknowledgment of the accuracy of information submitted.
+
+**Can I assign multiple Forms to a single asset?**
+
+You sure can! Please keep in mind that an Asset will only be considered Documented or Verified if all required questions are completed on all assigned Forms.
+
+### API Tutorials
```

Comment:
Maybe one more FAQ:

How do I know that a Form is being completed?

Great question. We are working on Compliance Forms Analytics that will directly show you the progress of your initiative across the selected assets. Stay tuned!

Reply:
This is very important.

Else people are like... So I created a form... great.... now what?

Reply:
We could also add an FAQ about "How the user knows about a Compliance Form assigned to them"? 

And the response is: They have to check the Inbox on the navigation bar. There are no off-platform notifications for compliance forms at this time.

Reply:
Very important we are as clear about what we DONT support as what we DO